### ISSUE TITLE
SVM

### ISSUE THREAD

**Issue by maytepenella** [jsantalo/happybirds#9]
Implement a SVM

**Comment by borbota** [comment]
I got worse results than with NB, but let's try.

**Comment by ldescampsvila** [comment]
Yes, me too. The teacher said that with SVM we need to introduce a Vector with less characteristics than RandomForest, if not, the result will be worst.

**Comment by maytepenella** [comment]
I reopen the issue because it could work if we use embedding for characteristic extraction and SVM as classifier. --> it is explained in reference about word2vec.

**Comment by maytepenella** [comment]
Add a optimization function in order to obtain best parameters for the model
